src/api/middleware/validators/offer.js

```diff
@@ -262,7 +282,9 @@ const edit = useExpressValidators([
         .optional()
         .isISO8601({ strict: true }).withMessage(ValidationReasons.DATE).bail()
         .isAfter().withMessage(ValidationReasons.DATE_EXPIRED).bail()
-        .custom(publishEndDateEditable),
+        .custom(publishEndDateEditable)
+        .custom(publishEndDateBeforePublishDate)
```

Comment:
This is unnecessary since publishEndDateEditable already does the check (and more - since it checks the current value of publishDate in the offer, for example)